src/QMCHamiltonians/PairCorrEstimator.cpp

```diff
@@ -102,6 +102,19 @@ PairCorrEstimator::PairCorrEstimator(ParticleSet& elns, std::string& sources)
 
 void PairCorrEstimator::resetTargetParticleSet(ParticleSet& P) {}
 
+// The value should match the index to norm_factor in set_norm_factor
+const int PairCorrEstimator::gen_pair_id(const int& ig, const int& jg)
+{
+  if (jg < ig)
+  {
+    return ((num_species * (num_species - 1)) / 2) - (((num_species - jg) * (num_species - jg - 1)) / 2) + ig;
```

Comment:
Please remove {} around one line and reduce () as well. Do the same for else as well.